egui_http_web/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "egui_http_web"
```

Comment:
If possible I'd prefer if you could combine `egui_http_web` and `egui_http_native` to one crate (`epi_http`) instead, with different dependencies and code paths depending on platform.

Reply:
okay i will change it